### PR TITLE
Count documents in collections

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Add documents count in collection list

image

### Other changes

 - Rename `Collections/List` to `Collections/CollectionList` to match component name

### CODE REVIEW

File: test/e2e/cypress/integration/collections.spec.js [hunk 1]
```diff
@@ -89,11 +89,14 @@ describe('Collection management', function() {
     cy.contains('This collection is empty')
   })
 
-  it('is able to delete a collection', function() {
+  it.only('is able to delete a collection', function() {
```
Comment (Njuelle) [inline test/e2e/cypress/integration/collections.spec.js:92]: you forgot this ;)

File: src/components/Data/Collections/CollectionList.vue [hunk 2]
```diff
@@ -292,38 +293,47 @@ export default {
...
-          name: c,
-          type: 'realtime'
-        })),
-        ...this.storedCollections.map(c => ({
-          name: c,
+    storedCollections() {
+      const rawStoredCollections = this.rawStoredCollections
+        ? this.rawStoredCollections
+        : this.$store.state.index.indexesAndCollections[this.index].stored
+      console.log({ rawStoredCollections})
```
Comment (Njuelle) [inline src/components/Data/Collections/CollectionList.vue:324]:
```suggestion
```

File: src/directives/Materialize/m-select.directive.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ export default {
     setTimeout(() => {
       $el.material_select()
       // $el.on('change', () => {
```
Comment (xbill82) [inline src/directives/Materialize/m-select.directive.ts:11]: Leftover?

Comment (Aschen) [reply src/directives/Materialize/m-select.directive.ts:11]: I didn't change this code

Comment (xbill82) [reply src/directives/Materialize/m-select.directive.ts:11]: k

**Pull request merged**